### ISSUE TITLE
Quadros e tabelas na lista de figuras e tabelas

### ISSUE THREAD

**Issue by rafaelalvesitm** [douglasrizzo/Classe-Latex-FEI#36]
Olá, percebi um problema ao utilizar o ambiente quadro e table junto com tabular e a formatação que aparece na lista de figuras e tabelas. 

Pelo que entendi do guia da biblioteca a lista de figuras pode conter além das figuras, quadros, mapas, diagramas etc. Enquanto a lista de tabelas deve conter apenas as tabelas (dados tratados estatisticamente, o que quer que isso signifique). Percebi que ao utilizar o ambiente quadro com tabular o quadro aparece na lista de figuras corretamente. Entretanto, a lista de tabelas tem a sua contagem conforme a ordem de tabelas e quadros que apareceram anteriormente. 

Para exemplificar, imagine a seguinte ordem no corpo do texto. 

Figura 
Quadro 
Figura 
Tabela 
Figura 
Quadro 
Tabela 

Atualmente a lista de figuras conteria a seguinte informação:
Figura 1
Quadro 1
Figura 2
Figura 3
Quadro 3

e a lista de tabelas conteria
Tabela 2
Tabela 4

Acredito que o correto seria:
Figura 1
Quadro 1
Figura 2
Figura 3
Quadro 2

e então
Tabela 1
Tabela 2

Espero que eu tenha conseguido evidenciar o erro e que eu tenha compreendido corretamente o guia da biblioteca.

**Comment by leonardoSCosta** [comment]
Sim, parece que ao criar um quadro o contador de tabelas também acaba sendo incrementado. O problema também aparece mesmo sem quadros, por exemplo, essa é a minha lista de tabelas, note que começa do número 2.

![image](https://user-images.githubusercontent.com/37155964/212142777-e2fe062e-f9ac-46f1-abf8-be3309d39636.png)

E na lista de figuras só existem figuras. Outro detalhe é que ele está pulando uma linha entre figuras de capítulos diferentes.

![image](https://user-images.githubusercontent.com/37155964/212142889-7eca5029-20ce-410c-a380-054063915af8.png)

Esse bug de pular as linhas parece que está acontecendo desde o commit https://github.com/douglasrizzo/Classe-Latex-FEI/commit/4e3c390352a2a3664e90ceb50edef59751c5af79, pelo menos é o que da pra perceber pelo `fei.pdf`. No commit https://github.com/douglasrizzo/Classe-Latex-FEI/commit/57e573c34ffc807fe0fdb796bea494159873ba5a ainda está normal.

**Comment by douglasrizzo** [comment]
Enviei uma correção do problema mencionado pelo @leonardoSCosta para a branch dev.

Ainda estou procurando o problema da numeração das tabelas.

**Comment by douglasrizzo** [comment]
Eu percebi que esse problema na numeração das tabelas está acontecendo no manual da classe, mas não consegui reproduzi-lo em um exemplo mínimo de debug que criei, com diferentes tipos de floats em diferentes ordens.

![image](https://user-images.githubusercontent.com/2209364/212543753-599d65ae-863c-49a0-8298-927c7ec65263.png)

![image](https://user-images.githubusercontent.com/2209364/212543762-90dcbc33-0de2-4cc8-84d5-8f2a15149ca9.png)

O problema não deve ser relacionado aos quadros, uma vez que todos os _floats_ personalizados (organogramas, desenhos, quadros, mapas, gráficos etc.) são definidos [da mesma forma](https://github.com/douglasrizzo/Classe-Latex-FEI/blob/43228e780566990fb67f12c744b327d5de77e5c3/fei.dtx#L1858-L1948), então, se um problema acontecesse com quadros, deveria acontecer com os outros floats.

[testa_numeracao_floats.zip](https://github.com/douglasrizzo/Classe-Latex-FEI/files/10420284/testa_numeracao_floats.zip)
